derp/derp_server.go

```diff
@@ -182,6 +186,12 @@ func NewServer(privateKey key.Private, logf logger.Logf) *Server {
 		memSys0:              ms.Sys,
 		watchers:             map[*sclient]bool{},
 		sentTo:               map[key.Public]map[key.Public]int64{},
+		bytes_pool: &sync.Pool{
```

Comment:
Pools are typically package globals in Go, not tied to the life of any other object.